### PR TITLE
Fixed overflow issue on product's page

### LINKED ISSUE #2292: Dot background breaking page on iOS browsers + desktop Chrome/Safari
On production, pages that have the dot background ex https://foundation.mozilla.org/en/privacynotincluded/products/apple-watch-4/ have black space to the right of the page (this makes voting on the slider hard).

Screen cap from iPhone:
image

Also noticing Related Products on these pages (pages with dot background) it much narrower than ones that don't.

With dot background:
image

Without dot background:
image

cc @Pomax @mmmavis

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #2292 

I added a few wrapper divs so it will be easier to review without whitespace/indentation changes: https://github.com/mozilla/foundation.mozilla.org/pull/2303/files?w=1

### CODE REVIEW

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 1]
```diff
@@ -39,11 +39,11 @@
   </div>
 </div>
 
-{% with section_class="p-md-5 col-12 col-lg-10 offset-lg-1" %}
+{% with section_class_col="col-12 col-lg-10 offset-lg-1" section_class_padding_x="px-md-5" section_class_padding_y="py-md-5" %}
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/product_page.html]: I had to break this into 3 vars as we can't override Bootstrap classes that are there for the same breakpoints

e.g., `p-md-5 p-md-0` --> `p-md-5` overrides `p-md-0`

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 2]
```diff
@@ -125,64 +125,66 @@ <h3 class="h3-heading h3-heading-small">What could happen if something went wron
...
-        <a class="d-block" href="/en/privacynotincluded/products/{{ product.slug }}">
-          {% if USE_CLOUDINARY %}
-            <img class="thumb-border"
-            width="300"
-            src="{% cloudinary_url product.cloudinary_image quality=50 fetch_format="auto" crop="fit" width=736 %}"
-            alt="{{product.name}}">
-          {% else %}
-            <img src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image }}" width="600">
+      {% if product.updates|length > 0 %}
+        <div class="{{section_class_col}} {{section_class_padding_x}} mt-5 bg-white">
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/product_page.html]: I mainly just changed classes on line 137 and 139

**Comment by kristinashu** [comment]
Awesome! I checked this one which has the dot background and it seems to work well.

**Comment by mmmavis** [comment]
@Pomax / @alanmoo you can check out this product for review

https://foundation-mofostaging-pr-2303.herokuapp.com/en/privacynotincluded/products/lot-artist/ 

(Product image might not show but I think that's loudinary's issue)

File: source/sass/buyers-guide/views/product.scss [hunk 3]
```diff
@@ -196,23 +196,13 @@
...
-  left: -100%;
-  right: -100%;
+  left: 0;
+  right: 0;
   height: 228px;
   background-image: url("/_images/buyers-guide/dot-pattern.svg");
   background-size: 9px 9px;
 }
 
-.row.with-product-updates {
```
Comment (alanmoo) [inline source/sass/buyers-guide/views/product.scss]: I think these might still be necessary; this section looks pretty narrow on mobile.

Comment (alanmoo) [reply source/sass/buyers-guide/views/product.scss]: (I'm not sure this is actually the solution, but my "request changes" is to fix that in general)

Comment (mmmavis) [reply source/sass/buyers-guide/views/product.scss]: Ummmm it's weird. It's exactly the same code but things look differently on my machine and on review app. I'll investigate.

# my machine
screen shot 2018-11-29 at 12 45 08

# Heroku review app
screen shot 2018-11-29 at 12 45 12

Comment (mmmavis) [reply source/sass/buyers-guide/views/product.scss]: Ah it's something to do with the "compare products" section. I'll update my PR.

Comment (mmmavis) [reply source/sass/buyers-guide/views/product.scss]: I think it's something to do with how other sections on this page are set up (e.g., with/without product image, compare products section etc). I'll take a closer look in a bit when I get to the office.

**Comment by mmmavis** [comment]
Will file a followup ticket and post the issue number here.

**Comment by mmmavis** [comment]
Follow-up ticket filed: #2308

**Pull request merged**